sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/automl_image.py

```diff
@@ -28,6 +28,21 @@ def __init__(
         sweep: Optional[ImageSweepSettings] = None,
         **kwargs,
     ) -> None:
+        """Base class for all AutoML Image jobs.
```

Comment:
`set_data` and `set_limits` docstrings are displayed incorrectly in the docs:
<img width="389" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/38021489/309a23cb-ba4e-4a59-bb7a-063c1723d674">

I think simply adding a newline before the param list will fix this

Reply:
fixed.